src/ecalc/libraries/libecalc/common/libecalc/input/validation/generate_json_schema.py

```diff
@@ -33,18 +33,58 @@ def get_template(schema: str, schema_name: str, is_root: bool = False) -> str:
     return template
 
 
+def replace_property_with_legacy_json_schema(schema: dict, property_key: str, property_ref: str) -> dict:
```

Comment:
for which JSON Schema version/revision does this apply to? Is it due to our requirements of an older version of the schema than what is recommended ?

Reply:
Ah, legacy_json_schema refers to our previous implementation (json-schema files), i.e. replace the asset property TIME_SERIES with a `$ref` to the `time-series.json` json-schema. So this is a temporary thing until we can create all the pydantic classes for yaml